### ISSUE TITLE
Question about low performance on AV2 (FastNSF)

### ISSUE THREAD

**Issue by jang-1996** [KTH-RPL/OpenSceneFlow#31]
Hello, thank you very much for your impressive work and for making the code publicly available.

I am currently running FastNSF, and I have a question regarding unexpectedly low performance.

While experimenting with the configuration, I observed the following behavior:
	•	When I increased early_patience in the config to 30, I could see a clear improvement in performance.
	•	However, when I further increased early_patience to 150, the performance no longer improved and remained low.

Based on my understanding of the Argoverse 2 (AV2) test dataset, a validation set performance around the 0.4 (Dynamic Bucket-Normalized EPE Mean) range seems reasonable, but my results do not consistently reach this level depending on the early_patience setting.

I was wondering if you could provide some guidance on:
	•	How early_patience is intended to be tuned for FastNSF
	•	Whether there are recommended values or best practices for AV2 validation set.
	•	Or if there are other hyperparameters that are particularly sensitive when optimizing on AV2

Thank you again for your great work, and I would really appreciate any advice or clarification.

Best regards.

<img width="441" height="256" alt="Image" src="https://github.com/user-attachments/assets/47d5e9ae-4012-4943-a0ea-ca7a19b90815" />

<img width="381" height="253" alt="Image" src="https://github.com/user-attachments/assets/93535dfb-0aba-4227-a60a-e8df3d9af5e2" />

<img width="420" height="254" alt="Image" src="https://github.com/user-attachments/assets/579e2532-ab58-404e-92fe-6dcf80d078ab" />
